lib/gstreamer/msdk/util.py

```diff
@@ -108,6 +108,9 @@ def mapprofile(codec,profile):
     "av1-8"  : {
       "main"                  : "main",
     },
+    "av1-10"  : {
+      "main"                  : "main-10",
```

Comment:
This should be `"main10" : "main-10"`.

Also, fwiw, `mapprofile` is not used by decoder tests.  It's only used in encoder tests.  Thus, this change is unnecessary for adding decode support in this patchset.

Reply:
updated by removing it